### PR TITLE
Add Martin Fink's GR (WS 22)

### PR DESCRIPTION
**Pull request by martin-fink**
### Checklist (To mark a task as complete, use `[x]`)
- [x] The year and semester of the archive directory are correct.
- [ ] A presentation PDF is in the `talks` folder.
- [x] A final report or a final thesis is in the `docs` folder.
- [x] README.md is updated and properly rendered on GitHub.
- [x] Each pdf size is less than 5MB.

### CODE REVIEW

File: archive/2022/summer/docs/gr_fink_dependencies-lk-backend.pdf
Comment (mmisono) [inline archive/2022/summer/docs/gr_fink_dependencies-lk-backend.pdf:1]: Should the file be in `archive/2022/winter`?

Comment (martin-fink) [reply archive/2022/summer/docs/gr_fink_dependencies-lk-backend.pdf:1]: Of course, thanks!

File: archive/2022/README.md [hunk 1]
```diff
@@ -28,4 +28,5 @@
...
 | Jiong Liu | Evaluation of Data-Intensive Accelerated Functions in Computational Storage Devices | BA | Charalampos Mainas, Atsushi Koshiba | [PDF](winter/docs/bsc_liu_evaluation_of_data-intensive_accelerated_functions_in_computational_storage_devices.pdf) | [PDF](winter/talks/bsc_liu_evaluation_of_data-intensive_accelerated_functions_in_computational_storage_devices.pdf) |
 | Jonas Zöschg                     | End-to-End On-Chip Encryption to Prevent Physical Attacks                                                     | BA   | Harshavardhan Unnibhavi                           | [PDF](winter/docs/bsc_zoeschg_end_to_end_on_chip_encryption_to_prevent_physical_attacks.pdf)                                 | [PDF](winter/talks/bsc_zoeschg_end_to_end_on_chip_encryption_to_prevent_physical_attacks.pdf) |
+| Martin Fink                      | Detecting Address Dependencies Broken by the LLVM Backend in the Linux Kernel       | GR   | Paul Heidekrüger, Pramod Bhatotia                 | [PDF](summer/docs/gr_fink_dependencies-lk-backend.pdf)                                                                       |  |
```
[the remark this answers is not in the document]
Comment (martin-fink) [reply archive/2022/README.md]: Oh wow looks like I should go to sleep instead 🙂

**Review by mmisono — APPROVED** [review]

**Comment by mmisono** [comment]
(I think we do not need to separate the directory for summer and winter to simplify management)

**Pull request merged**